NitroxClient/MonoBehaviours/DiscordRP/AcceptRequest.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections;
+using NitroxClient.MonoBehaviours.DiscordRP;
+using UnityEngine;
+
+namespace NitroxClient.MonoBehaviours.Gui.MainMenu
+{
+    public class AcceptRequest : MonoBehaviour
+    {
+        public DiscordController DiscordRp;
+        public DiscordRpc.DiscordUser Request;
+
+        Rect discordWindowRect = new Rect(Screen.width / 2 - 250, 100, 500, 250);
```

Comment:
This window is in the center of the screen but this means that anyone can annoy you very easily by spamming invites. I think it's better to move the request window to a corner of the screen so that it is less distracting.